packages/lib/availability.ts

```diff
@@ -0,0 +1,126 @@
+import dayjs, { ConfigType } from "dayjs";
```

Comment:
Moved here since the seeder also uses this logic and to avoid circular dependencies.